content/en/docs/reference/access-authn-authz/mutating-admission-policy.md

```diff
@@ -0,0 +1,54 @@
+---
+reviewers:
+- deads2k
+- sttts
+- cici37
+title: Mutating Admission Policy
+content_type: concept
+---
+
+<!-- overview -->
+
+{{< feature-state state="alpha" for_k8s_version="v1.32" >}}
+
+This page provides an overview of Mutating Admission Policy.
+
+<!-- body -->
+
+## What is Mutating Admission Policy?
+
+Mutating admission policies offer a declarative, in-process alternative to mutating admission webhooks.
+
+Mutating admission policies use the Common Expression Language (CEL) to declare mutations to resources.
+Mutations can be defined either with an *apply configuration* that is merged using the
+[server side apply merge strategy](https://kubernetes.io/docs/reference/using-api/server-side-apply/#merge-strategy)
+or a [JSON patch](https://jsonpatch.com/).
+
+Mutating admission policies are highly configurable, enabling policy authors to define policies
+that can be parameterized and scoped to resources as needed by cluster administrators.
+
+## What Resources Make a Policy
+
+A policy is generally made up of three resources:
+
+- The `MutatingAdmissionPolicy` describes the abstract logic of a policy
```

Comment:
Per the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/), drop the backticks.